python/pysrc/kaskada/sources/arrow.py

```diff
@@ -445,7 +446,8 @@ async def create(
         """Create a Parquet source.
 
         Args:
-            path: The path to the Parquet file to add.
+            path: The path to the Parquet file to add. This can be relative to the current
```

Comment:
I wonder if we should just require these be URLs (so `file://`)

Reply:
Resolving so I can merge but tagging @epinzur , thoughts?

Reply:
probably should be urls so we can add support to load from object storage by just changing the url (and maybe setting some additional auth params)

Reply:
Agreed with Eric that these should be URLs. Also should probably move to a different file (`_parquet.py`) since this is no longer using PyArrow under the hood to load the data in memory.